app/scripts/templates/sign_in.mustache

```diff
@@ -64,7 +64,7 @@
       {{^suggestedAccount}}
         <form novalidate>
         <div class="input-row">
-          <input type="email" class="email{{#email}} prefilled{{/email}}" placeholder="{{#t}}Email{{/t}}" value="{{ email }}" spellcheck="false" {{^email}}autofocus{{/email}}>
+          <input name="email" type="email" class="email{{#email}} prefilled{{/email}}" placeholder="{{#t}}Email{{/t}}" value="{{ email }}" spellcheck="false" {{^email}}autofocus{{/email}}>
```

Comment:
We still need `value={{email}}` here. The email can also come from the relier, and [we decide which email to use in js](https://github.com/mozilla/fxa-content-server/blob/dff0aeac9166beb2af3c8af5994f22590145a666/app/scripts/views/sign_in.js#L78). Same goes for the SignUp view. I tried to figure out a way around this, and was unable to come up with anything clean.